src/layerstack.stanza

```diff
@@ -389,7 +391,7 @@ When used on its own, this function is useful for creating asymmetric
 pcb stackups. This function is most useful when used on a non-empty
 layer stack.
 
-@param l-set layer or set of layers to add to the top of the stack. Note
+@param l-set layer or set of layers to add to the top of the stack, ordered from inner to outer (top to bottom). Note
```

Comment:
Same comment here - I think this is confusing. Let's re-word this as: 

`ordered from inner-most to outer-most on the bottom side`